### ISSUE TITLE
Fix issues with persisting or retrieving settings

### ISSUE THREAD

**Issue by dagolap** [Smeedee/Smeedee#61]
In Trondheim we have experienced a lot of issues seemingly related to persisting (or perhaps retrieving?) settings.

This leads to issues where we have to configure timespans for working days left, top committers, commit statistics etc. every time we restart Smeedee. Once that is done however, all data is transferred and shown properly, so the problem only seems to exist with regards to settings values.

**Comment by dagolap** [comment]
Also, this goes for what builds should be filtered out from the Build Status Widget etc.

All user connections etc are good as well, so I'm fairly certain we are only talking about settings values.

Note that effor should be undertaken so see whether this is an issue with retrieving or storing the settings.

**Comment by benteca** [comment]
The mentioned Widgets are fixed. Closing this. If there are problems with other widgets, we can create a new issue for them.
